r/rsparkling/DESCRIPTION

```diff
@@ -9,10 +9,11 @@ Authors@R: c(
   person("Jakub", "Hava", role = c("aut"), email = "jakub@h2o.ai"),
   person("RStudio", role = c("cph")),
   person("H2O.ai", role = c("cph")))
-Depends:
+Imports:
     utils,
     sparklyr (>= 0.3),
     h2o (>= 3.8.2.9)
```

Comment:
If we are moving those deps from Depends to Imports I will also need to update `r/examples/example_rsparkling.R`, so PR in current state is not yet ready to merge.